src/model/codacyInput.ts

```diff
@@ -39,3 +39,11 @@ export class Codacyrc {
     this.tools = tools
   }
 }
+
+export class Specification {
```

Comment:
What you mean here by `Specification`.
It seems that you're duplicating the already existing `Patterns` class